### PR TITLE
Updated the project to integrate with passport

### PR DESCRIPTION
**Pull request by larce1212**

### CODE REVIEW

File: api/controllers/AuthController.js [hunk 1]
```diff
@@ -1,175 +1,35 @@
...
-   * @param   {Request}   request     Request object
-   * @param   {Response}  response    Response object
-   */
-  authenticated: function authenticated(request, response) {
-    if (request.isAuthenticated()) {
-      response.json(200, request.user);
-    } else {
-      response.json(200, false);
+var passport = require('passport');
+
```
Comment (wolf-mtwo) [inline api/controllers/AuthController.js]: how 4 spaces? use only 2

Comment (larce1212) [inline api/controllers/AuthController.js]: Looks like more of the source code are with 4 spaces

Comment (larce1212) [inline api/controllers/AuthController.js]: We will work in a script to format all the source code with 2 spaces in the identation

Comment (wolf-mtwo) [inline api/controllers/AuthController.js]: jajja as you wish sir

**Comment by vico-dev-z** [comment]
An issue will be created to have a proper indentation, so for now, merging the code.

**Pull request merged**